.cirrus.yml

```diff
@@ -129,16 +131,17 @@ gating_task:
     # in sync at all times.
     vendor_script:
         - '/usr/local/bin/entrypoint.sh .install.vndr'
-        - '/usr/local/bin/entrypoint.sh vendor'
-        - 'cd /go/src/github.com/containers/libpod && ./hack/tree_status.sh'
+        - 'cd $GOSRC && make vendor && ./hack/tree_status.sh'
 
     # This task builds Podman with different buildtags to ensure the build does
-    # not break.
+    # not break.  It also verifies all sub-commands have man pages.
     build_script:
-        - '/usr/local/bin/entrypoint.sh clean podman'
-        - '/usr/local/bin/entrypoint.sh clean podman-remote'
-        - '/usr/local/bin/entrypoint.sh clean podman BUILDTAGS="exclude_graphdriver_devicemapper selinux seccomp"'
-        - '/usr/local/bin/entrypoint.sh clean podman-remote-darwin'
+        - '/usr/local/bin/entrypoint.sh podman'
+        - 'cd $GOSRC && ./hack/podman-commands.sh'
+        # N/B: entrypoint.sh resets $GOSRC (same as make clean)
+        - '/usr/local/bin/entrypoint.sh podman-remote'
+        - '/usr/local/bin/entrypoint.sh podman BUILDTAGS="exclude_graphdriver_devicemapper selinux seccomp"'
```

Comment:
This and podman-remote are the main ones that I think require a `clean` - we need to wipe files that are inside the repository (commited parts of the codebase) and regenerate them.

Reply:
oooohhh, okay, good enough.  I'll put them back in.